### DISCUSSION TITLE
HELP ME PULSANTE 2 SECONDI

### DISCUSSION THREAD

**Discussion by giangre** [RavenSystem/esp-homekit-devices#2332] [MEPLHAA Scripts Collection]
HO UNO SHELLY 1 GEN3 VOLEVO INSERIRE UNA STRINGA COL COMANDO PULSANTE ATTIVO PER 2 SECONDI COSA DEVO INSERIRE NEL MELPHAA SCRIPT
GRAZIE

**Comment by RavenSystem** [comment]
```json
{"c":{"io":[[[5,19],2],[[1],6,1],[[10],6,0,1],[[3],10,0,3]],"l":19,"b":[[1,5]]},"a":[{"i":2,"0":{"r":[[5]]},"1":{"r":[[5,1]]},"b":[[10],[10,0]],"es":[{"t":22,"h":2,"n":5,"g":3,"j":10,"y0":[{"v":75,"r":1,"0":{"m":[[1]]}}]}]}]}
```

**Reply by giangre** [reply]
Ho provato non dà l'impulso di apertura e chiusura contatto dai contatti OI lo Shelly 13g si accende e si spegne ma il contatto dello 0I DEVE ESSERE IN CONTATTO CHIUSO OVVERO DEVE FARE CONTATTO NEI SECONDI DI ACCENSIONE DELLO SHELLY
